scripts/generate-goreleaser/generate-goreleaser.go

```diff
@@ -0,0 +1,143 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	_ "embed"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/goreleaser/goreleaser/pkg/config"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/yaml.v2"
+)
+
+const dockerBaseImageTemplateName = "docker.io/persesdev/perses"
+
+//go:embed .goreleaser.base.yaml
+var baseConfig []byte
+var (
+	dockerSupportedArches            = []string{"amd64", "arm64"}
+	mapDockerfileNameAndTemplateName = map[string][]string{
+		"Dockerfile":                  {"", "distroless"},
+		"distroless-debug.Dockerfile": {"debug", "distroless-debug"},
+	}
+)
+
+func join(elems []string, sep string) string {
+	var sb strings.Builder
+	for i := 0; i < len(elems); i++ {
+		el := elems[i]
+		if len(el) == 0 {
+			continue
+		}
+		sb.WriteString(el)
+		if i < len(elems)-1 {
+			nextEl := elems[i+1]
+			if len(nextEl) == 0 {
+				continue
+			}
+			sb.WriteString(sep)
+		}
+	}
+	return sb.String()
+}
+
+func readGoreleaserBaseFile() *config.Project {
+	c := &config.Project{}
+	if err := yaml.Unmarshal(baseConfig, c); err != nil {
+		logrus.Fatal(err)
+	}
+	return c
+}
+
+func generateDockerConfig(c *config.Project) {
+	var binaryIDs []string
+	for _, build := range c.Builds {
+		binaryIDs = append(binaryIDs, build.ID)
+	}
+	for dockerfileName, templateNames := range mapDockerfileNameAndTemplateName {
+		for _, arch := range dockerSupportedArches {
+			c.Dockers = append(c.Dockers, config.Docker{
+				Goos:       "linux",
+				Goarch:     arch,
+				IDs:        binaryIDs,
+				Dockerfile: dockerfileName,
+				Use:        "buildx",
+				ImageTemplates: []string{
+					fmt.Sprintf("%s:{{ .Tag }}-%s-%s", dockerBaseImageTemplateName, templateNames[1], arch),
+				},
+				BuildFlagTemplates: []string{
+					"--pull",
+					"--label=org.opencontainers.image.authors=The Perses Authors <perses-team@googlegroups.com>",
```

Comment:
I completely forgot we had a google group (it is in the governance docs, but I haven't looked at those in aaaages). Is that something I should get added to as a maintainer?

Reply:
yup you should be added ;)